lib/ex_webrtc/rtp_transceiver.ex

```diff
@@ -132,14 +146,18 @@ defmodule ExWebRTC.RTPTransceiver do
 
     codecs = get_codecs(mline, config)
     rtp_hdr_exts = get_rtp_hdr_extensions(mline, config)
-    sender = RTPSender.update(transceiver.sender, mid, List.first(codecs), rtp_hdr_exts)
+    codec = List.first(codecs)
```

Comment:
In theory, the other side may send data using any payload format negotiated in SDP offer/answer. IIRC, they can even switch between codecs during the session without renegotiation. I don't want to handle that behaviour but maybe we should somewhere ensure that  a packet we receive has expected payload type?

Reply:
got it https://www.rfc-editor.org/rfc/rfc7160#section-4.1